ckanext/iaea/templates/snippets/organization.html

```diff
@@ -0,0 +1,6 @@
+{% ckan_extends %}
+{% block info %}
+    {% block package_openness %}
+        {{ super() }} 
```

Comment:
Why are we overriding this template? Does it had conflicts with something else? Because as it is right now is just does whatever the previous block had defined (calling just super, but not adding anything else)

Reply:
It's a template tweak so when ckanext-qa is installed it shows the openness score of the datasets. It will extend from here:
https://github.com/keitaroinc/ckanext-qa/blob/master/ckanext/qa/templates/snippets/organization.html.